src/Chainweb/Pact/Service/Types.hs

```diff
@@ -71,6 +71,7 @@ type PactExMVar t = MVar (Either PactException t)
 data NewBlockReq = NewBlockReq
     { _newBlockHeader :: BlockHeader
     , _newMiner :: Miner
+    , _newCreationTime :: BlockCreationTime
```

Comment:
`StrictData` is on here right? I don't know why we turn it on, because the reviewer is always going to ask "is there a bang on here"

Reply:
StrictData isn't on in this module at all.